server/routes/activities/record-attendance/handlers/removePay.test.ts

```diff
@@ -127,7 +127,7 @@ describe('Route Handlers - Remove Pay', () => {
         .calledWith(1, res.locals.user)
         .mockResolvedValue({
           id: 1,
-          date: format(new Date(), 'yyyy-MM-dd'),
+          date: '2024-01-12',
```

Comment:
This test started failing as the error message validation was checking for 12/01/2024.